src/callstack/index.ts

```diff
@@ -74,7 +74,7 @@ class CallstackHeader extends Widget {
     super({ node: document.createElement('header') });
 
     const layout = new PanelLayout();
-    const span = new Widget({ node: document.createElement('span') });
+    const span = new Widget({ node: document.createElement('h2') });
```

Comment:
Maybe this should be called `title` like the other one.

Reply:
Yes that's better.